lib/app/features/posts/view/favorites_view.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:black_tax_and_white_benefits/app/features/posts/data/favorites_repository.dart';
+import 'package:black_tax_and_white_benefits/app/features/posts/view/post_cell.dart';
+import 'package:flutter/material.dart';
+import 'package:go_router/go_router.dart';
+import 'package:hooks_riverpod/hooks_riverpod.dart';
+
+class FavoritesView extends StatelessWidget {
+  const FavoritesView({super.key});
+
+  @override
+  Widget build(BuildContext context) => Scaffold(
+        appBar: AppBar(title: const Text('Favorites')),
+        body: Center(
+          child: Consumer(
+            builder: (context, ref, child) {
+              final posts = ref.watch(favoriteListProvider);
+              debugPrint(posts.valueOrNull?.map((e) => '${e.id}').toString());
```

Comment:
the map and functional programming is cool right

Reply:
Yeah